debian/control

```diff
@@ -34,7 +34,9 @@ Build-Depends:
    python3-lxml,
    python3-defusedxml,
    python3-xmlschema (>= 1.10.0),
-   python3-elementpath (>= 2.5.0)
+   python3-elementpath (>= 2.5.0),
```

Comment:
@junjiemao1 @NanlinXie The elementpath version must be = 2.5.0 and not greater, right?

Reply:
If it needs to be exactly a version, the Debian control file syntax is: `python3-elementpath (= 2.5.0)`

Reply:
OK (the pip install syntax is with ==).  Thanks for clarifying for the control file.  We'll need @junjiemao1 to confirm the exact version (and I'll update the GSG documentation as well).

Reply:
As this PR is against the main branch, preserving the `>= 2.5.0` is also fine to me. I have a patch to resolve the incompatibility issue so that any version since 2.5.0 (including the latest 2.5.3) will work.